test/rules/github.js

```diff
@@ -138,3 +138,105 @@ test.serial('github - remote origin is an GitLab repo', async t => {
 		}
 	]);
 });
+
+test.serial('github - invalid token', async t => {
+	const execaStub = sandbox.stub(github.execa, 'stdout');
+	const gotStub = sandbox.stub(github.got, 'get');
+
+	execaStub
+		.withArgs('git', ['config', '--get', 'remote.origin.url'])
+		.returns('git@github.com:sindresorhus/awesome-lint-test.git');
+
+	gotStub
+		.withArgs('https://api.github.com/repos/sindresorhus/awesome-lint-test')
+		.rejects({
+			statusCode: 401
+		});
+
+	const messages = await lint({config, filename: 'test/fixtures/github/0.md'});
+	t.deepEqual(messages, [
+		{
+			ruleId: 'awesome/github',
+			message: 'Unauthorized access or token is invalid'
+		}
+	]);
+});
+
+test.serial('github - API rate limit with token', async t => {
```

Comment:
```suggestion
test.serial('github - API rate limit exceeded with token', async t => {
```

Seems to me like a more descriptive name to this test (as well as the other similar tests).